easybuild/easyconfigs/b/Blender/Blender-2.79b-foss-2018b-Python-3.6.6-CUDA-9.2.88.eb

```diff
@@ -0,0 +1,77 @@
+easyblock = 'CMakeMake'
+
+name = 'Blender'
+version = '2.79b'
+cuda_ver = '9.2.88'
+versionsuffix = '-Python-%%(pyver)s-CUDA-%s' % cuda_ver
+
+homepage = 'https://www.blender.org/'
+description = """Blender is the free and open source 3D creation suite. It supports
+ the entirety of the 3D pipeline-modeling, rigging, animation, simulation, rendering,
+ compositing and motion tracking, even video editing and game creation."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['http://download.blender.org/source/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['4c944c304a49e68ac687ea06f5758204def049b66dc211e1cffa1857716393bc']
+
+# disable SSE detection to give EasyBuild full control over optimization compiler flags being used
+configopts = '-DWITH_CPU_SSE=OFF -DCMAKE_C_FLAGS_RELEASE="-DNDEBUG" -DCMAKE_CXX_FLAGS_RELEASE="-DNDEBUG" '
+
+# these are needed until extra dependencies are added for them to work
+configopts += '-DWITH_INSTALL_PORTABLE=OFF '
+configopts += '-DWITH_BUILDINFO=OFF '
+configopts += '-DWITH_GAMEENGINE=OFF '
+configopts += '-DWITH_SYSTEM_GLEW=OFF '
+
+# Python paths
+site_packages = '$EBROOTPYTHON/lib/python%(pyshortver)s/site-packages'
+configopts += '-DPYTHON_VERSION=%(pyshortver)s '
+configopts += '-DPYTHON_LIBRARY=${EBROOTPYTHON}/lib/libpython%%(pyshortver)sm.%s ' % SHLIB_EXT
+configopts += '-DPYTHON_INCLUDE_DIR=${EBROOTPYTHON}/include/python%(pyshortver)sm '
+configopts += '-DPYTHON_NUMPY_PATH=' + site_packages + '/numpy-1.15.0-py%(pyshortver)s-linux-x86_64.egg '
+configopts += '-DPYTHON_REQUESTS_PATH=' + site_packages + '/requests-2.19.1-py%(pyshortver)s.egg '
```

Comment:
the numpy and requests paths are not needed for Blender itself but for addons that depend on them. Blender installs its own version of Python (+extensions) and complains if you don't provide the full paths.

Reply:
to be more precise, Blender looks for:
.../site-packages/numpy
.../site-packages/requests
which are obviously not present.

it would indeed be better to avoid hard-coding the version numbers, but I could not find an elegant way to do that. maybe you have an idea?